### DISCUSSION TITLE
Individual voice chat audio level for each player

### DISCUSSION THREAD

**Discussion by Sanel-V** [FakeFishGames/Barotrauma#8398] [Ideas and suggestions]
While playing, you might come across many different players with varying microphone quality and loudness.
While one player might be very loud, another might be very quiet. 

Asking each person to increase or decrease their mic input volume isn't always feasible, since it might impact that players voice activation settings, or other players volume settings (just because it might be loud/quiet to me, might not be to someone else).
Decreasing/increasing incoming voice volume globally isn't a good option either, since it would make other player's quieter/louder voice even quieter/louder.

An option to individually adjust incoming voice of players seperately would be greatly appreciated.

**Comment by Diagonir** [comment]
Old thread but still relevant today, one person is incredibly loud and the other incredibly low and it makes the whole voice chat a whole lot less appealing

**Reply by Arctodites** [reply]
This is already a feature by now.  You just have to click on the 'manage'  option in the pop-up menu on the player's character name in the Crew List, and from there you can adjust the volume of his or her voice chat.

**Comment by NotWendy** [comment]
Maybe also an option to disable the extra compression radio chat adds to the player's voice? Some people have bad mics to begin with.

**Comment by Regalis11** [comment]
This has been implemented since this was posted, closing
